Assessment

besperspektivnyak opened the pull request:
Начальная версия модуля по оценке диалогов. Пока ничего не возращает, но проверяет на наличие перебиваний, обсценной лексики и слов-паразитов.

besperspektivnyak committed:
Delete init.txt

besperspektivnyak committed:
Upload assessment.py and search_words.py

Добавление файлов с оценкой диалогов

MaEgV committed:
Update assessment.py

Функция оценки по критерию теперь возвращает имя файла с результатами по каждому критерию
Общей оценки пока нет(нет инсайдов)

MaEgV committed:
Update assessment.py

proted reviewed (commented):
Постарайся в дальнейшем писать более содержательные комментарии к коммитам

MaEgV committed:
Update search_words.py

MaEgV committed:
Update assessment.py

MaEgV committed:
Create parasites.txt

MaEgV committed:
Create obscene.txt

besperspektivnyak committed:
Add files via upload

Фразы вынесены в отдельный файл .py. Добавилась функция поиска слов. Добавлен параметр границы, числа допустимых слов.

besperspektivnyak committed:
Delete phrases.py

besperspektivnyak committed:
Add files via upload

Оценка идет по большему числу слов.

besperspektivnyak committed:
Обновление списка обсценной лексики

besperspektivnyak committed:
Добавлена функция вывода оценки (?)

besperspektivnyak committed:
add interrupting file
